Arduino-Bling/RobotBling/animations.ino

```diff
@@ -0,0 +1,37 @@
+/*
+ * Robot animations
+ */
+
+/*
+ * Solid colors - remember to add 'delay()' to listen for new commands
+ */
+
+void solid(rgbColor c)
+{
+  setAllPixels(c.color);
+  showPixels();
+  delay(10);
+}
+
+void solidWithFullBrightness(rgbColor c, uint8_t brightness)
+{
+  setFullBrightness();
+  solid(c);
+}
+
+void solidWithBlink(rgbColor c, uint8_t wait)
+{
+  solid(c);
+  blinkOffWithDelay(wait);
```

Comment:
I hadn't thought of this design pattern - delay, then turn off, then delay, and allow the animation to automatically restart. That's pretty neat.

Reply:
cool!